From efe908afd97346ff3e3efff87700bb07d46e7e08 Mon Sep 17 00:00:00 2001
From: Samuel Karp <skarp@amazon.com>
Date: Fri, 26 Mar 2021 17:05:21 -0700
Subject: [PATCH 4/5] bottlerocket: remove unsupported CNI plugins

Bottlerocket is enabling support for awsvpc mode, but not AppMesh yet.
---
 agent/app/agent_unix.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/agent/app/agent_unix.go b/agent/app/agent_unix.go
index 0d8ae66..bc3daed 100644
--- a/agent/app/agent_unix.go
+++ b/agent/app/agent_unix.go
@@ -46,7 +46,6 @@ const initPID = 1
 var awsVPCCNIPlugins = []string{ecscni.ECSENIPluginName,
 	ecscni.ECSBridgePluginName,
 	ecscni.ECSIPAMPluginName,
-	ecscni.ECSAppMeshPluginName,
 	ecscni.ECSBranchENIPluginName,
 }
 
-- 
2.32.0

